### ISSUE TITLE
About the loss.backward() error reporting

### ISSUE THREAD

**Issue by Shaoyi0520** [traveller59/spconv#462]
Hello. I recently encountered a problem with spconv2.x. I set batch_size to 20 and the GPU has 5G of video memory left, but when I increase the batch_size, loss.backward() reports an error, I don't know why this is, can you suggest to me some possible reasons? Thank you very much.
![image](https://user-images.githubusercontent.com/60254694/162609766-d0544548-ee34-4ee2-a8b4-b8380f36de43.png)
